### ISSUE TITLE
missing icons for sound and network

### ISSUE THREAD

**Issue by duck7000** [shimmerproject/elementary-xfce#416]
After installing elementary xfce icons (or git version, no difference) i noticed that the sound icon is missing.
This is in al variants
![Screenshot at 2023-12-20 21-06-23](https://github.com/shimmerproject/elementary-xfce/assets/15360048/f33e01e7-d67b-402e-adea-05ee770dbabc)

in elementary xfce and elementary xfce darkest the network icon is missing
![Screenshot at 2023-12-20 21-09-05](https://github.com/shimmerproject/elementary-xfce/assets/15360048/778b41be-8049-410e-939f-76c24df613f5)

This is tested on Manjaro Mate and endeavourOS Mate

Edit: i checked with those icons but with a different theme and the icons are there! but they are almost the same color as the background, so on a black panel bar it is hardly noticed. On a light panel bar it is fine but not a black one like with the matcha theme

**Comment by ignaciop** [comment]
Try selecting "elementary Xfce dark" in System Settings -> Appearance -> Icons.

In think elementary Xfce darkest is deprecated in 0.18

**Comment by duck7000** [comment]
Yes i know darkest is deprecated, in the other versions it still don't work right

I want to use the dark version, but in the dark version the sound icon is the wrong color with a black panel bar

On my other machine i have manjaro xfce with both elementary-icons (dependency of xfce) and elementary-xfce-icons installed and the sound icon is working fine so it may have something to do with elementary-icons installed or not

**Comment by ignaciop** [comment]
I download directly v0.18 and unzipped in /usr/share/icons (Debian). Hope it helps!

**Comment by duck7000** [comment]
@ignaciop Installed it, no change unfortunately

**Comment by newhoa** [comment]
Thanks for reporting!

The reason we can't fix this by making the volume icon light is... Say we switch to a light volume icon so it can be seen in dark panels. Now in light windows and menus, that icon is no longer visible. By trying to "fake" symbolic icons with monochrome icons in certain places, it makes them not easy or impossible to see in other places.

Probably the best way to fix this is to get rid of the monochrome status icons and colorize them so they are visible on light and dark backgrounds. We've talked about this before and should probably start moving toward that. The Wifi icon was recently updated to be that way, and hopefully is a good solution for that.

Symbolic icons (like what is used for the Xfce Panel Pulse Audio Plugin) are a solution to this because they will match the font color (light on dark panels, dark on light panels).

MATE developers are not interested in implementing symbolic icons. [Here is a solution they suggest](https://github.com/mate-desktop/caja/issues/1723#issuecomment-1605846775), which is forcing symbolic icons in the Gtk Theme gtk.css:

> ```
> /*mate-volume-control*/
> /*A symbolic icon is used here and will be gray without this*/
> GvcMixerDialog image{
> 	-gtk-icon-style: symbolic;
> }
> ```

Another suggestion might be to use the Xfce panel on the MATE DE.

Anyway, hope this helps even if it isn't a very straightforward solution/answer.
